azure_media_services/__init__.py

```diff
@@ -18,11 +19,20 @@
     sys.modules['edxval'] = mock.MagicMock()
     sys.modules['edxval.models'] = mock.MagicMock()
 
-    sys.modules['openedx'] = mock.MagicMock()
-    sys.modules['openedx.core'] = mock.MagicMock()
-    sys.modules['openedx.core.djangoapps'] = mock.MagicMock()
-    sys.modules['openedx.core.djangoapps.lang_pref'] = mock.MagicMock()
-    sys.modules['openedx.core.djangoapps.lang_pref.api'] = mock.MagicMock()
+    sys.modules['util'] = mock.MagicMock()
+    sys.modules['util.views'] = mock.MagicMock()
 
+    sys.modules['opaque_keys'] = mock.MagicMock()
+    sys.modules['opaque_keys.edx'] = mock.MagicMock()
+    sys.modules['opaque_keys.edx.keys'] = mock.MagicMock()
+
+    sys.modules['xmodule'] = mock.MagicMock()
+    sys.modules['xmodule.modulestore'] = mock.MagicMock()
+    sys.modules['xmodule.modulestore.django'] = mock.MagicMock()
+
+    settings.configure(
+        LMS_ROOT_URL='http://lms.com'
```

Comment:
Shouldn't this be read from configuration?

Reply:
I think, not possible since, there is no platform dependency here.